src/e2e-test/features/servicenowsource/RunTime.feature

```diff
@@ -67,4 +67,37 @@ Feature: ServiceNow Source - Run time scenarios
     And Wait till pipeline is in running state
     And Open and capture logs
     And Verify the pipeline status is "Succeeded"
-    And Verify count of no of records transferred to the target BigQuery Table
\ No newline at end of file
+    And Verify count of no of records transferred to the target BigQuery Table
+
+  @TS-SN-RNTM-03 @CONNECTION @SN_SOURCE_CONFIG @SN_RECEIVING_SLIP_LINE @BQ_SINK
+  Scenario: Verify user should be able to deploy and run the pipeline using connection manager functionality
+    When Open Datafusion Project to configure pipeline
+    And Select plugin: "ServiceNow" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "ServiceNow"
+    And Click plugin property: "switch-useConnection"
+    And Click on the Browse Connections button
+    And Click on the Add Connection button
+    And Click plugin property: "connector-ServiceNow"
+    And Enter input plugin property: "name" with value: "connection.name"
+    And fill Credentials section for pipeline user
+    Then Click on the Test Connection button
+    And Verify the test connection is successful
+    Then Click on the Create button
+    And Use new connection
+    And configure ServiceNow source plugin for table: "RECEIVING_SLIP_LINE" in the Table mode
+    Then Validate "ServiceNow" plugin properties
+    And Close the Plugin Properties page
+    And Select Sink plugin: "BigQueryTable" from the plugins list
+    And Connect source as "ServiceNow" and sink as "BigQuery" to establish connection
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Replace input plugin property: "project" with value: "projectId"
+    And Enter input plugin property: "datasetProject" with value: "datasetprojectId"
+    And Configure BigQuery sink plugin for Dataset and Table
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Wait till pipeline is in running state
+    And Open and capture logs
+    And Verify the pipeline status is "Succeeded"
+    Then Verify count of no of records transferred to the target BigQuery Table
```

Comment:
If possible, can we also add a feature step at the end to verify if the values of the records at the source side is same as the values of records received in the BQ sink table side? This will ensure that the data is not getting corrupted in between because of any datatype conversions.

Reply:
Actually there is a release planned next week for this Plugin with Connection Management Functionality. So had to add the e2e tests for connection management in the develop first. The Data Validation is done for Service Now Sink e2e as it is a recent Plugin but Service Now Source and Multi Source don't have a step yet.

Is that a possibility that we can merge the e2e tests for connection management first and Data Validation for Source and Multi Source tests can be added to Backlog.  As soon as we are done with the releases next week I will take up this task on my own. @MrRahulSharma @Aryan-Verma What do you think ?

Reply:
Sure, works for me

Reply:
Works for me as well.

Reply:
Thanks, Can we go for a merge if these changes look fine.